notebooks/concepts/Specviz from splot concept notebook.ipynb

```diff
@@ -0,0 +1,3636 @@
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Fitting: function\n",
+    "\n",
+    "*Description*:\n",
+    "\n",
+    "Function   to   be  fit  to  the  spectra.   The  functions  are \n",
+    "        \"legendre\"   (legendre   polynomial),   \"chebyshev\"   (chebyshev  \n",
+    "        polynomial),  \"spline1\"  (linear  spline),  and \"spline3\" (cubic\n",
+    "        spline).  The functions may be abbreviated.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Fitting: order\n",
+    "\n",
+    "*Description*:\n",
+    "\n",
+    "**order**\n",
+    "        The order of the polynomials or the number of spline pieces."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Fitting: low_reject, high_reject\n",
+    "\n",
+    "*Description*:\n",
+    "\n",
+    "Rejection limits below  and  above  the  fit  in  units  of  the\n",
+    "        residual  sigma.   Unequal  limits  are  used to reject spectral\n",
+    "        lines on one side of the continuum during continuum fitting.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Fitting: niterate\n",
+    "\n",
+    "*Description*:\n",
+    "\n",
+    "Number of rejection iterations."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Fitting: grow\n",
+    "\n",
+    "*Description*:\n",
+    "\n",
+    "When a pixel is rejected, pixels within  this  distance  of  the\n",
+    "        rejected pixel are also rejected.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Fitting: markrej\n",
+    "\n",
+    "*Description*:\n",
+    "\n",
+    "Mark  rejected  points?   If  there  are many rejected points it\n",
+    "        might be desired to not mark rejected points."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Arithmetic-related operations "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This section deals with arithmetic between multiple spectra.\n",
+    "\n",
+    "1. In the notebook, this is trivial, as `Spectrum1D` objects support arithmetic.  E.g., subtracting a smoothed version of a spectrum from the spectrum and showing it can be done as:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "specviz.load_data(specviz.current_spectrum - box_smooth(specviz.current_spectrum, 10))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "2. In the UI, this is potentially much more complicated, as a plugin to support arithmetic would be necessary. Such a plugin is possible, but not necessarily desirable, as part experience suggests the feature requests for this quickly spiral, suggesting simply saying \"use the notebook\" is a better answer.\n",
```

Comment:
I still think this is important to have in the UI (desktop, MAST).  Our users ranked this highly as a feature they wanted to have in some form in the quicklook.  I think it's fine to adopt a default, simple, and transparent methodology for doing this in the UI with a disclaimer saying, "here's how this works, for more control, please use the notebook".

Reply:
Hmm.  As noted above I've reworded this to allow room for there to be a distinct arithmetic plugin if we can carefully avoid feature creep. I'm thinking maybe we should make a separate issue to track this question though.  @havok2063, do you think you can make that, specifically with the MAST use case listed as the first point? (all of the use cases I know of for the desktop tools I can tell people "use the notebook" but I acknowledge that's a bit harder for MAST quicklook).

Reply:
Yeah I can do that.  Should this be a Github ticket or JIRA ticket?  I could add this to the list of tickets in the JIRA epic.  In my mind the core MAST user workflow is " let me quickly view and analyze in a simple and dirty fashion to determine if this file/target/object merits downloading and further analysis, so I don't waste my time".  In the MAST case where we only load a single spectrum, then I think there's not much need for an arithmetic plugin.  But in cubeviz or the MAST case of loading multiple spectra, simple arithmetic would be desired.